doc/source/templates/testing/docker/04_finetuning_llms_with_deepspeed/requirements.txt

```diff
@@ -14,4 +14,5 @@ protobuf<3.21.0
 torchmetrics
 lm_eval==0.3.0
 tiktoken==0.1.2
-sentencepiece
\ No newline at end of file
+sentencepiece
+peft @ git+https://github.com/huggingface/peft.git@08368a1
```

Comment:
can we have the full git commit string, rather than just 7 chars?

Reply:
Yep!